app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -14,96 +15,105 @@ import TableMetricPrimary from "./TableMetricPrimary";
 import TableMetricSecondary from "./TableMetricSecondary";
 import TableResults from "./TableResults";
 
-const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
-  <AppLayoutWithExperiment
-    title="Analysis"
-    testId="PageResults"
-    analysisRequired
-    redirect={({ status, analysis }) => {
-      if (!status?.released) {
-        return "edit/overview";
-      }
+const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
+  const { outcomes: configOutcomes } = useConfig();
 
-      if (analysisUnavailable(analysis)) {
-        // Return to the experiment root/summary page
-        return "";
-      }
-    }}
-  >
-    {({ experiment, analysis }) => {
-      // For testing - users will be redirected if the analysis is unavailable
-      // before reaching this return, but tests reach this return and
-      // analysis.overall is expected to be an object (EXP-800)
-      if (analysisUnavailable(analysis)) return <></>;
+  return (
+    <AppLayoutWithExperiment
+      title="Analysis"
+      testId="PageResults"
+      analysisRequired
+      redirect={({ status, analysis }) => {
+        if (!status?.released) {
+          return "edit/overview";
+        }
 
-      const slugUnderscored = experiment.slug.replace(/-/g, "_");
-      return (
-        <>
-          <LinkMonitoring {...experiment} />
-          <h3 className="h5 mb-3 mt-4" id="overview">
-            Overview
-          </h3>
-          <p className="mb-4">
-            Detailed analysis{" "}
-            <LinkExternal
-              href={`https://protosaur.dev/partybal/${slugUnderscored}.html`}
-              data-testid="link-external-results"
-            >
-              can be found here
-            </LinkExternal>
-            .
-          </p>
-          <h3 className="h6">Hypothesis</h3>
-          <p>{experiment.hypothesis}</p>
-          <TableHighlights
-            primaryProbeSets={experiment.primaryProbeSets!}
-            results={analysis!}
-            {...{ experiment }}
-          />
-          <TableHighlightsOverview
-            {...{ experiment }}
-            results={analysis?.overall!}
-          />
+        if (analysisUnavailable(analysis)) {
+          // Return to the experiment root/summary page
+          return "";
+        }
+      }}
+    >
+      {({ experiment, analysis }) => {
+        // For testing - users will be redirected if the analysis is unavailable
+        // before reaching this return, but tests reach this return and
+        // analysis.overall is expected to be an object (EXP-800)
+        if (analysisUnavailable(analysis)) return <></>;
```

Comment:
Looks like this was moved from existing code, but this should maybe just be a `return` or a `return null`? Or a `return ""` like earlier in the file